mm2src/lp_network.rs

```diff
@@ -137,6 +138,16 @@ async fn process_p2p_message(
                 lp_swap::process_msg(ctx.clone(), split.next().unwrap_or_default(), &message.data).await;
                 to_propagate = true;
             },
+            Some(lp_swap::TX_HELPER_PREFIX) => {
+                if let Some(pair) = split.next() {
+                    if let Ok(Some(coin)) = lp_coinfind(&ctx, pair).await {
+                        match coin.send_raw_tx_bytes(&message.data).compat().await {
+                            Ok(id) => log::info!("Transaction broadcasted successfully: {:?} ", id),
```

Comment:
I'm afraid it can be annoying to log this info with the `LogLevel::Info` level given that such P2P messages will be sent during every swap. How about logging with the `LogLevel::Debug` level?

Reply:
Sounds better, what you think ? @artemii235

Reply:
Sounds good to me 🙂